### DISCUSSION TITLE
how to set focus on highlighted text?

### DISCUSSION THREAD

**Discussion by 6shruthi5** [IgniteUI/igniteui-angular#9052] [General]
IgxTextHighlightDirective : From many days I am trying to set focus on the highlighted text with no success.
my content is large and hence user cannot scroll the page to see next highlighted text so I was wondering if there is any way I can set focus so that on click on next button page will automatically scroll to the found searach term.

https://stackblitz.com/edit/texthighlight-shr-wp3lna?file=src%2Fapp%2Ftext-highlight-sample-1%2Ftext-highlight-sample-1.component.ts

**Comment by gedinakova** [comment]
Hello, I suggest you get the active highlighted element and call scrollIntoView() as shown in the updated [sample application](https://stackblitz.com/edit/texthighlight-shr-7a8wny?file=src%2Fapp%2Ftext-highlight-sample-1%2Ftext-highlight-sample-1.component.ts). 
```ts
document.querySelector('.igx-highlight--active').scrollIntoView();
```
Please, let us know if you need further assistance on the matter.
